Packs/LogPoint_SIEM_Integration/Integrations/LogPoint_SIEM_Integration/LogPoint_SIEM_Integration.yml

```diff
@@ -719,16 +732,14 @@ script:
     - contextPath: LogPoint.SearchLogs
       description: Search results.
       type: String
-  dockerimage: demisto/python3:3.10.14.90585
+  dockerimage: demisto/python3:3.10.14.92207
   isfetch: true
   runonce: false
-  script: '-'
+  script: ''
   subtype: python3
   type: python
 tests:
 - LogPoint SIEM Integration - Test Playbook 1
 - LogPoint SIEM Integration - Test Playbook 2
 - LogPoint SIEM Integration - Test Playbook 3
-defaultmapperin: LogPoint SIEM Integration - Incoming Mapper
-defaultclassifier:
```

Comment:
Were these removed intentionally?

Reply:
Hi Anas, sorry for the late return, I've been on holiday for the last 10 days. 

The removing is not intentionally, we can re-add it.

Thank you for your review.

Reply:
Please re-add them.

Reply:
Re-added